dsc/cli.py

```diff
@@ -105,16 +105,7 @@ def post_main_group_subcommand(
 def reconcile(ctx: click.Context) -> None:
     """Reconcile bitstreams with item identifiers from the metadata."""
     workflow = ctx.obj["workflow"]
-    no_bitstreams, no_item_identifiers = workflow.reconcile_bitstreams_and_metadata()
-
-    if no_bitstreams:
-        logger.error(f"No bitstreams found for these item identifiers: {no_bitstreams}")
-    if no_item_identifiers:
-        logger.error(
-            f"No item identifiers found for these bitstreams: {no_item_identifiers}"
-        )
-    else:
-        logger.info("All item identifiers and bitstreams successfully matched")
+    workflow.reconcile_bitstreams_and_metadata()
```

Comment:
Any messages are logged within the implemented `reconcile_bitstreams_and_metadata` (and any of its submethods) for a given workflow.